portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyTestFile.path

```diff
@@ -11,38 +11,38 @@
 
 <tbody>
 <tr>
-	<td>OPEN_SIMULATION</td>
-	<td>//button[@data-qa-id='simulation']</td>
+	<td>APPLY_CUSTOM_SIZE</td>
+	<td>//button[@aria-label='Apply Custom Size']</td>
 	<td></td>
 </tr>
 <tr>
-	<td>CUSTOM_SCREEN</td>
-	<td>//span[text()='Custom']</td>
+	<td>ASSERT_VALUES</td>
+	<td>//div[text()='Custom Size 1920x1080 Applied']</td>
 	<td></td>
 </tr>
 <tr>
-	<td>CUSTOM_HEIGHT</td>
-	<td>//input[@id='_com_liferay_product_navigation_simulation_web_portlet_SimulationPortlet_height']</td>
+	<td>CLOSE_BUTTON</td>
+	<td>//button[contains(@class,'btn btn-monospaced') and @aria-label='Close']</td>
 	<td></td>
 </tr>
 <tr>
-	<td>CUSTOM_WIDTH</td>
-	<td>//input[@id='_com_liferay_product_navigation_simulation_web_portlet_SimulationPortlet_width']</td>
+	<td>CUSTOM_HEIGHT</td>
+	<td>//input[@id='_com_liferay_product_navigation_simulation_web_portlet_SimulationPortlet_height']</td>
```

Comment:
Eu li e entendi o seu comentario em relacao a essa path, mas como ficaria em um exemplo desses? Pq se voce for reparar bem, oq diferencia a "height" da "width" é justamente a ultima palavra do id e pelo oq eu olhei os outros atributos eram muito genericos, a respeito dessa tag html "input".